runtime/oti/j9generated.h

```diff
@@ -39,6 +39,10 @@ typedef struct J9SidecarExitFunction {
 	void (*func)(void);
 } J9SidecarExitFunction;
 
+#if defined(J9VM_OPT_MICROJIT)
+typedef struct J9JITConfig J9MicroJITConfig;
```

Comment:
What's the value of this definition? Why not just use `struct J9JITConfig`?

Reply:
I think this was a hold-over from the original attempt to port the old MicroJIT. I've removed it and replaced usage with the `struct J9JITConfig` as suggested.

Reply:
The only remaining change to this file is the copyright date: please revert that - no changes are required here.